### DISCUSSION TITLE
Nuxt v2 - Successful deploy via CLI, but not GitHub?

### DISCUSSION THREAD

**Discussion by entity** [vercel/community#373] [Help]
Hi all,

I've deployed my project to Vercel from the Vercel CLI, it deploys fine, I then go and link my GitHub repo. I try deploy from GitHub, but then get the following error:

Any reason why?
```
Error: No Output Directory named "dist" found after the Build completed. You can configure the Output Directory in your Project Settings. Learn More: https://vercel.link/missing-public-directory
```

**Comment by samcx** [comment]
Is it possible to share with us the Deployment Overview URL?
